libmupen64plus/mupen64plus-input-bkm/mupen64plus-input-bkm/mupen64plus-input-bkm.vcxproj

```diff
@@ -72,15 +73,15 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <LinkIncremental>false</LinkIncremental>
-    <OutDir>$(SolutionDir)..\..\..\..\output\dll\</OutDir>
+    <OutDir>$(SolutionDir)..\..\..\output\dll\</OutDir>
     <PostBuildEventUseInBuild>true</PostBuildEventUseInBuild>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <OutDir>$(SolutionDir)..\..\..\..\output\dll\</OutDir>
     <PostBuildEventUseInBuild>true</PostBuildEventUseInBuild>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
-    <OutDir>$(SolutionDir)..\..\..\..\output\dll\</OutDir>
+    <OutDir>$(SolutionDir)..\..\..\output\dll\</OutDir>

```

Comment:
Should be `Assets\dll`—and please change this across the rest of Mupen too.

Reply:
This change will only fix this file. Others are out of scope for this pull.